runner/src/server/index.ts

```diff
@@ -105,7 +105,6 @@ async function createServer(routeConfig: RouteConfig) {
   await server.register(configureCrumbPlugin(config, routeConfig));
   await server.register(Schmervice);
   await server.register(pluginAuth);
-  await server.register(clientSideUpload);
```

Comment:
just out of curiosity, why do we remove this line and add it in 173 instead?

Reply:
Good point! I'll add a comment clarifying this.

https://github.com/communitiesuk/digital-form-builder/blob/3d0d6fddc5e5c7a3e505409b91a4a06f2c60dd81/runner/src/server/index.ts#L174-L176